cvat/apps/engine/views.py

```diff
@@ -420,6 +438,15 @@ def annotations(self, request, pk):
         else:
             return Response("Format is not specified",status=status.HTTP_400_BAD_REQUEST)
 
+    @action(methods=['GET'], detail=True, url_path='backup')
+    def export(self, request, pk=None):
+        db_project = self.get_object() # force to call check_object_permissions
+        return backup.export(db_project, request)
+
+    @action(detail=False, methods=['POST'])
+    def backup(self, request, pk=None):
```

Comment:
@azhavoro , It looks like instead of `POST /api/v1/tasks/backup` we should have `POST /api/v1/tasks` with a corresponding data type. Also I don't like that for projects we create an empty project and after that import a dataset and for backup with have another approach. For creating tasks we have also two steps: create a tasks and send data after that.

Let's keep it as is for now and try to find a common solution.
@Marishka17 , please look at the inconsistency. It is something which we need to solve during REST API redesign.